packages/dapp/src/actionCreators/network.ts

```diff
@@ -1,12 +1,24 @@
 import { AnyAction } from "redux";
+import { Subscription } from "rxjs";
 
 export enum networkActions {
   SET_NETWORK = "SET_NETWORK",
 }
 
-export const setNetwork = (network: string): AnyAction => {
+const internalSetNetwork = (network: string): AnyAction => {
   return {
     type: networkActions.SET_NETWORK,
     data: network,
   };
 };
+
+export const setNetwork = (network: string): any => {
+  return (dispatch: any, getState: any): AnyAction => {
+    const { listingHistorySubscriptions } = getState().networkDependent;
+    listingHistorySubscriptions.forEach((subscription: Subscription, key: string) => {
```

Comment:
shouldn't the linter complain about the unused argument?

Reply:
it doesn't seem to, and i'm going to leave it as is because it makes it more clear that it's a map instead of a list/set. Although it's absolutely crazy that it's (value, key) and not (key, value)